requirements.txt

```diff
@@ -8,7 +8,7 @@ click
 cloup
 humanfriendly
 tabulate
-tenacity
+tenacity<8.4.0
```

Comment:
For 8.4.0 tenacity build failed. BTW python3.6 for the newest lib versions out of support: https://tenacity.readthedocs.io/en/latest/changelog.html#upgrade-notes  
```
env PATH=venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin PYTHON_VERSION=3.10 PYTHONIOENCODING=utf8 CLICKHOUSE_VERSION=latest COMPOSE_HTTP_TIMEOUT=300 python -m tests.integration.env_control create
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/data/disk1/home/mburdukov/workspace/ch-backup/tests/integration/env_control.py", line 12, in <module>
    from tests.integration.modules import compose, docker, minio, templates
  File "/data/disk1/home/mburdukov/workspace/ch-backup/tests/integration/modules/minio.py", line 9, in <module>
    from tenacity import retry, retry_if_exception_type, stop_after_attempt, wait_fixed
  File "/data/disk1/home/mburdukov/workspace/ch-backup/venv/lib/python3.10/site-packages/tenacity/__init__.py", line 653, in <module>
    from tenacity.asyncio import AsyncRetrying  # noqa:E402,I100
ModuleNotFoundError: No module named 'tenacity.asyncio'
make: *** [Makefile:167: .session_conf.sav] Error 1
```